docs/source/notes/autograd.rst

```diff
@@ -245,49 +261,42 @@ where
+The key assumption made in the derivation of the above formula is that function :math:`F` is a part of a
+real valued function. For more details, please check out Section 3.4 `here <https://arxiv.org/pdf/1701.00392.pdf>`_.
 
-    .. math:: \begin{bmatrix} c & -d \end{bmatrix} * J * \begin{bmatrix} 1 \\ -1j \end{bmatrix}
+**What happens for cross-domain functions?**
+*********************************************
 
-In PyTorch, the `VJP` is mostly what we care about, as it is the computation performed when we do backward
-mode automatic differentiation. Notice that d and :math:`1j` are negated in the formula above. Please look at
-the `JAX docs <https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#Complex-numbers-and-differentiation>`_
-to get explanation for the negative signs in the formula.
+The above formula can also be verified to be correct for cross-domain functions functions.
 
-**What happens if I call backward() on a complex scalar?**
-*******************************************************************************
+The :math:`VJP` for a function :math:`F: ℂ → ℝ`:
 
-The gradient for a complex function is computed assuming the input function is a holomorphic function.
-This is because for general :math:`ℂ → ℂ` functions, the Jacobian has 4 real-valued degrees of freedom
-(as in the `2x2` Jacobian matrix above), so we can’t hope to represent all of them with in a complex number.
-However, for holomorphic functions, the gradient can be fully represented with complex numbers due to the
-Cauchy-Riemann equations that ensure that `2x2` Jacobians have the special form of a scale-and-rotate
-matrix in the complex plane, i.e. the action of a single complex number under multiplication. And so, we can
-obtain that gradient using backward which is just a call to `vjp` with covector `1.0`.
+    .. code::
 
-The net effect of this assumption is that the partial derivatives of the imaginary part of the function
-(:math:`v(x, y)` above) are discarded for :func:`torch.autograd.backward` on a complex scalar
-(e.g., this is equivalent to dropping the imaginary part of the loss before performing a backwards).
+        def F(z):
+            x, y = real(z), imag(z)
+            s = u(x, y)
+            return s
 
-For any other desired behavior, you can specify the covector `grad_output` in :func:`torch.autograd.backward` call accordingly.
+at :math:`x + yj` can be simplified to be written as:
 
-**How are the JVP and VJP defined for cross-domain functions?**
-***************************************************************
+    .. math:: real(grad_out) * \frac{\partial s}{\partial z^{*}}
 
-Based on formulas above and the behavior we expect to see (going from :math:`ℂ → ℝ^2 → ℂ` should be an identity),
-we use the formula given below for cross-domain functions.
+Note that the Wirtinger and Conjugate Wirtinger derivative in this case are conjugate of each other.
```

Comment:
I am not sure that this note is important here and it might be confusing with the point we make above when phrased this way.

Reply:
point taken and removed!